### PR TITLE
OKTA-460712 Update Implement grant type docs to accommodate new stack-selector and OIE org link

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add nutrition.md snippets to Implement authorization by grant type docs to customize the stack-selector options and to add OIE org link for Interaction Code.
- **Is this PR related to a Monolith release?** No

### Netlify preview:

https://620175d8a3c5e5490a000aeb--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/authcode/main/

### Resolves:

* [OKTA-460712](https://oktainc.atlassian.net/browse/OKTA-460712)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+**Learning outcomes**
+
+* Understand the OAuth 2.0 Authorization Code flow.
+* Set up your app with the Authorization Code grant type.
+* Implement the Authorization Code flow in Okta.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md:13]: We could remove the URL in the link, and replace with (/signup) -- as in the Note that comes after.

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/nutrition.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+**Learning outcomes**
+
+* Understand the OAuth 2.0 Authorization Code with PKCE flow.
+* Set up your app with Authorization Code and Refresh Token grant types.
+* Implement the Authorization Code with PKCE flow in Okta.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/nutrition.md:13]: Same comment for all as above.

**Review by brianduffield-okta — APPROVED** [review]
Hey Van, just a few nits.

Also suggest that we change the heading and nutrition facts code samples (nut-facts-samplecode.md) file (for those flows that have it) to "Web application examples" for clarity. And maybe just include the link rather than "See Examples." in the snippet, which I think looks a little awkward. It's not explicitly stated in Christine's nutrition facts site, so, open to debate or interpretations.

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md:13]: I thought we used the full URL for the /signup page to open a new tab? I'll confirm. Thanks

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/nutrition.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+**Learning outcomes**
+
+* Understand the Interaction Code flow.
+* Set up your app with the Interaction Code grant type.
+* Implement the Interaction Code flow in Okta.
+
+**What you need**
+
+* [Okta Identity Engine Developer Edition org](https://developer.okta.com/signup/oie-preview.html)
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/nutrition.md:13]: Good 👀 ! I'll didn't realize this and will follow up.

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md:13]: I spoke with Susan about this convention: most of our /signup links are full URLs to open a new tab (with the exception of some quick-start guides and the ApiAmProdWarning snippet note). I'll follow the full URL convention for now. Thanks for pointing this out :-)

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcode/nutrition.md:13]: No problem with this. I like opening the new tab as well.

**Comment by brianduffield-okta** [comment]
LGTM :shipit:

**Pull request merged**